QUEST_LV_0100.tsv

```diff
@@ -1877,16 +1877,16 @@ QUEST_LV_0100_20150317_001872	$Oh, I didn't know I could count on you.{nl}I will
 QUEST_LV_0100_20150317_001873	{memo X}Explained about the writings on the epitaph.
 QUEST_LV_0100_20150317_001874	Historian Odell
 QUEST_LV_0100_20150317_001875	There was an order to withdraw from here.{nl}I still have many things to research on.
-QUEST_LV_0100_20150317_001876	$Are you looking for a person?{nl}Can you tell me the name of the person you are looking for?
+QUEST_LV_0100_20150317_001876	$Are you looking for someone?{nl}Can you tell me the name of the person you are looking for?
 QUEST_LV_0100_20150317_001877	$Are you sure the name is Rexipher?{nl}There's no one with that name on our list.{nl}You could try asking the other people to see if they know this Rexipher.
 QUEST_LV_0100_20150317_001878	I searched the list and asked other people..{nl}There's no one with the name Rexipher.{nl}I am sorry that I couldn't be a help.
 QUEST_LV_0100_20150317_001879	$Rexipher? I am not sure.{nl}Are you sure that's the correct name?
-QUEST_LV_0100_20150317_001880	$He must have a lot of courage if he moves around with the name Rexipher. 
+QUEST_LV_0100_20150317_001880	$He must have a lot of courage if he runs around with the name Rexipher. 
 QUEST_LV_0100_20150317_001881	$Hold on for a minute. I've seen this before from a book.{nl}Wasn't Rexipher a Demon Lord who is part of the army of the Demon King, Vaiga?
-QUEST_LV_0100_20150317_001882	Using the name of a demon, that doen't look like a good intent.
-QUEST_LV_0100_20150317_001883	$Rexipher?{nl} The one with silver hair and wearing a robe?
-QUEST_LV_0100_20150317_001884	$He is a crazy person who can control monsters.{nl}He made me like this and also destroyed the altar which I used for research.{nl}
-QUEST_LV_0100_20150317_001885	$First, go to the Chesed Altar. {nl}They are also aiming for the Gedulah Altar.
+QUEST_LV_0100_20150317_001882	Using the name of a demon, that doesn't look like they have good intentions.
+QUEST_LV_0100_20150317_001883	$Rexipher?{nl} The one with silver hair, wearing a robe?
+QUEST_LV_0100_20150317_001884	$He is a crazy person who can control monsters.{nl}He's the reason I'm wounded like this, and he also destroyed the altar which I used for research.{nl}
+QUEST_LV_0100_20150317_001885	$He's attacking Chesed Altar, go there first. {nl}He is also going for Gedulah Altar.
```

Comment:
I was going off of the quest text, what does the original say?  The quest text mentions that he's attacking it, so I'd like to change that too if it's not accurate.